authorization/src/lib.rs

```diff
@@ -60,19 +57,19 @@ impl AuthContract {
         e.contract_data().set(DataKey::Admin, admin);
     }
 
-    // Saves data that corresponds to an Identifier, with that Identifiers authorization
-    pub fn save_data(e: Env, auth: Signature, nonce: BigInt, num: BigInt) {
+    /// Save the number for an authenticated [Identifier].
+    pub fn save_num(e: Env, auth: Signature, nonce: BigInt, num: BigInt) {
         let auth_id = auth.get_identifier(&e);
 
         check_auth(
             &e,
-            &WrappedAuth(auth),
+            &NonceForSignature(auth),
             nonce.clone(),
-            Symbol::from_str("save_data"),
+            symbol!("save_num"),
```

Comment:
For constant symbols the macro is preferred as it performs the conversion from string to Symbol at compile time.